screens/home/HomeFeed.tsx

```diff
@@ -0,0 +1,103 @@
+import {
+  Button,
+  Center,
```

Comment:
```suggestion
  Box,
```
Native components automatically take up the full width that they can in a flexbox, and this is the behavior that we want for Posts. Although it works, `Center`ing them can lead to unexpected behavior and shrinkage.